style.css

```diff
@@ -4,6 +4,57 @@ body {
 
 ul {
   list-style-type: none;
-  margin: 10px 0;
+  margin: 10px auto;
   padding: 0;
+  border: 0.5px solid;
+  width: 80%;
+  max-width: 500px;
+}
+
+li:nth-child(odd) {
+  background: #f3f3f3;
+}
+
+.book {
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```